pipeline/transformer/transformer_miro/src/main/scala/uk/ac/wellcome/platform/transformer/miro/Main.scala

```diff
@@ -1,36 +1,78 @@
 package uk.ac.wellcome.platform.transformer.miro
 
+import scala.concurrent.ExecutionContext
+
 import akka.actor.ActorSystem
 import akka.stream.ActorMaterializer
+
+import org.scanamo.DynamoFormat
+
 import com.typesafe.config.Config
 import uk.ac.wellcome.json.JsonUtil._
-import uk.ac.wellcome.messaging.sns.NotificationMessage
-import uk.ac.wellcome.messaging.typesafe.{MessagingBuilder, SQSBuilder}
+
 import uk.ac.wellcome.models.work.internal.TransformedBaseWork
 import uk.ac.wellcome.platform.transformer.miro.services.{
   MiroTransformerWorkerService,
   MiroVHSRecordReceiver
 }
-import uk.ac.wellcome.platform.transformer.miro.source.MiroRecord
-import uk.ac.wellcome.storage.typesafe.S3Builder
 import uk.ac.wellcome.typesafe.WellcomeTypesafeApp
 import uk.ac.wellcome.typesafe.config.builders.AkkaBuilder
+import uk.ac.wellcome.platform.transformer.miro.source.MiroRecord
+import uk.ac.wellcome.platform.transformer.miro.models.MiroMetadata
 
-import scala.concurrent.ExecutionContext
+import uk.ac.wellcome.bigmessaging.typesafe.BigMessagingBuilder
+import uk.ac.wellcome.messaging.sns.NotificationMessage
+import uk.ac.wellcome.messaging.typesafe.SQSBuilder
+import uk.ac.wellcome.messaging.sns.SNSConfig
+
+import uk.ac.wellcome.storage.store.dynamo.{DynamoHashStore, DynamoHybridStore}
+import uk.ac.wellcome.storage.store.s3.S3TypedStore
+import uk.ac.wellcome.storage.store.HybridIndexedStoreEntry
+import uk.ac.wellcome.storage.{ObjectLocation, ObjectLocationPrefix}
+import uk.ac.wellcome.storage.typesafe.{DynamoBuilder, S3Builder}
+import uk.ac.wellcome.storage.dynamo.DynamoHashEntry
+import uk.ac.wellcome.storage.streaming.Codec._
 
 object Main extends WellcomeTypesafeApp {
+
+  type IndexEntry = HybridIndexedStoreEntry[ObjectLocation, MiroMetadata]
+
+  type HashEntry = DynamoHashEntry[String, Int, IndexEntry]
+
   runWithConfig { config: Config =>
+    // TODO: from where do we get the correct values for this?
```

Comment:
The `path` can be empty; the `namespace` is the name of the S3 bucket where you store the objects. Look for an S3-related flag in the current config?